ui/components/Navigator.js

```diff
@@ -104,8 +104,8 @@ const styles = (theme) => ({
   itemCategory: {
     backgroundColor: '#263238',
     boxShadow: '0 -1px 0 #404854 inset',
-    paddingTop: 16,
-    paddingBottom: 16,
+    paddingTop: 21,
+    paddingBottom: 21,
```

Comment:
Looks good, but please ensure to define metrics. We typically use rem units, so please adjust accordingly.